test/release/examples/primers/classes.chpl

```diff
@@ -140,39 +138,27 @@ proc printSum(arg: borrowed C) {
 }
 printSum(own); // same as printSum(own.borrow())
 
-//
-// There are a few method names that cause the method to have special
-// meaning.  A method named ``this`` allows a class to be indexed like
-// an array.  A method named ``these`` allows a class to be iterated
-// over.
-//
-class ArrayLike {
-  var a, b, c, d: int;
-  proc this(i:int) ref {
-    select i {
-      when 1 do return a;
-      when 2 do return b;
-      when 3 do return c;
-      when 4 do return d;
-      otherwise halt("index out of bounds", i);
-    }
-  }
+// A variable of class type cannot store ``nil`` unless it is
+// declared to have nilable class type. To create a nilable class
+// type, apply the ``?`` operator to another class type
+var x: borrowed C?; // default-initializes to ``nil``
 
-  iter these() ref {
-    yield a;
-    yield b;
-    yield c;
-    yield d;
-  }
-}
+// An expression non-nilable class type can implicitly convert
+// to a the related nilable class type.
```

Comment:
This sentence is weird. Maybe rephrase as: "Non-nilable class types can be implicitly converted to the corresponding nilable class type."